azure-workload-identity-webhook.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: azure-workload-identity-webhook
+  version: 1.5.0
+  epoch: 1
```

Comment:
```suggestion
  epoch: 0
```

Reply:
this is resolved via https://github.com/wolfi-dev/os/pull/50559/commits/375840479d3e90616a26e196033d959633011e8a